addons/binding/org.openhab.binding.amazonechocontrol/src/main/java/org/openhab/binding/amazonechocontrol/internal/WebSocketConnection.java

```diff
@@ -1,597 +1,597 @@
-/**
```

Comment:
You should normally not be touching this file, any idea what happened?

Reply:
Nope and now i try to figure out how to role back and exclude it.

Reply:
Think i am on track now :)